.github/workflows/cmake.yml

```
@@ -62,3 +77,10 @@ jobs:
       with:
         name: windows-mingw64
         path: build/libutf8proc.*
+    - name: Test Consuming
+      run: |
+        cmake --install build --prefix tmp/install
+        cmake -S test/app -B test/app/build -DCMAKE_INSTALL_PREFIX=tmp/install -G'MSYS Makefiles'
+        cmake --build test/app/build
+        export PATH="$PATH:$(pwd)/tmp/install/bin"
```

* We don't need `export` here because `PATH` is always exported.
* `$(pwd)/tmp/install/bin` should be the first entry of `$PATH` to avoid referring unintentional DLLs.

```suggestion
        PATH="$(pwd)/tmp/install/bin:$PATH"
```